mmaction/models/recognizers/recognizer_omni.py

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from typing import Dict, Sequence, Union
+
+import torch
+from mmengine.model import BaseModel
+
+from mmaction.registry import MODELS
+from mmaction.utils import ConfigType, ForwardResults, SampleList
+
+
+@MODELS.register_module()
+class RecognizerOmni(BaseModel):
+    """"""
```

Comment:
To make it easier for users to understand, please add docstring to describe the main difference between RecognizerOmni and BaseRecognizer.